service/src/main/kotlin/fi/espoo/evaka/application/PendingDecisionEmailService.kt

```diff
@@ -97,6 +99,15 @@ GROUP BY application.guardian_id
         return jobCount
     }
 
+    fun getFromAddress(lang: Language, env: Environment): String {
+        val fromAddress = env.getRequiredProperty("mail_reply_to_address")
+        val foo = env.getRequiredProperty("mail_sender_name")
+        return when (lang) {
+            Language.sv -> "$foo <$fromAddress>"
+            else -> "$foo <$fromAddress>"
+        }
+    }
```

Comment:
This whole method isn't used anywhere?

Reply:
Hmm it seems I didn't go through the commit at all... Sorry :pensive: